src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -104,7 +106,15 @@
             }
 
             if (isFormatInsideWhiteList(filePath, config.getFormats())) {
-                FileInfo currentFileInfo = generateFileInfo(config.getRepoRoot(), filePath);
+                FileInfo currentFileInfo;
+
+                try {
+                    currentFileInfo = generateFileInfo(config.getRepoRoot(), filePath);
+                } catch (InvalidPathException ipe) {
+                    logger.warning(String.format(INVALID_FILE_PATH_MESSAGE_FORMAT, filePath));
+                    continue;
+                }
+
                 setLinesToTrack(currentFileInfo, fileDiffResult);
                 fileInfos.add(currentFileInfo);
             }
```

Comment:
Not sure if the `continue` in the `catch` block is necessary.

Try simplify it like this?

```
if (isFormatInsideWhiteList(filePath, config.getFormats())) {    
    try {
        // generateFileInfo may throw InvalidPathException 
        FileInfo currentFileInfo = generateFileInfo(config.getRepoRoot(), filePath);
        setLinesToTrack(currentFileInfo, fileDiffResult);
        fileInfos.add(currentFileInfo);
    } catch (InvalidPathException ipe) {
        logger.warning(String.format(INVALID_FILE_PATH_MESSAGE_FORMAT, filePath));
    }
}
```